src/DynamoCore/Extensions/ExtensionLoader.cs

```diff
@@ -122,5 +124,10 @@ private void Log(string msg)
         {
             Log(LogMessage.Info(msg));
         }
+
+        /// <summary>
+        /// An event that is raised when an extension starts loading.
+        /// </summary>
+        public event Action<IExtension> ExtensionLoading;
```

Comment:
I think standard practice is to name an `event` with a verb. I know this seems dumb, but having conventions make it much easier to work on a code base.

Reply:
you mean with the verb first?
`LoadingExtension`?

Reply:
Eh, forget it. It's probably good.